app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml

```diff
@@ -50,7 +50,7 @@ $canApplyMsrp = $helper->isShowBeforeOrderConfirm($product) && $helper->isMinima
                                 <?php if (isset($_formatedOptionValue['full_view'])) :?>
                                     <?= $block->escapeHtml($_formatedOptionValue['full_view']) ?>
                                 <?php else :?>
-                                    <?= $block->escapeHtml($_formatedOptionValue['value'], ['span', 'a']) ?>
+                                    <?= $block->escapeHtml($_formatedOptionValue['value'], ['span']) ?>
```

Comment:
Is this change also related to the current bugfix? Just want to clarify the reason for changing item options rendering upon fixing the Qty input behavior.

Thank you.

Reply:
No.. I have reverted it.